src/services/service_manager.cpp

```diff
@@ -36,19 +34,10 @@ void ServiceManager::init() {
         setLogLevelViaEnv = true;
     }
 
-    logger->header("  _ _  ______  __     __");
-    logger->header("  __  /_  __/ / /__  / /_");
-    logger->header("   ___ / /_  / / _ \\/ __/");
-    logger->header("  __  / / /_/ /  __/ /_");
-    logger->header(" _ _ /_/\\____/\\___/\\__/");
```

Comment:
Forgot I removed this, I can add it back if you want to save it. I hardcoded logging to be off for now so it won't display if I leave it.